src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -143,6 +149,7 @@ public InMoov(String n) {
   transient public Arduino neopixelArduino;
   transient public UltrasonicSensor ultrasonicSensor;
   transient public ProgramAB chatBot;
+  transient public HtmlFilter htmlFilter;
```

Comment:
Why do you need this a member method?  if all you are doing is invoking the static strip html, you should not have a handle to it as a member to this class.